pkg/vpcmodel/nodesExplainability.go

```diff
@@ -115,15 +115,15 @@ func (c *VPCConfig) getVsiNode(name string) Node {
 // GetNodesWithinAddress is given a string address in CIDR format or exact IP address format, and
 // returns the list of all internal nodes with this address
 func (c *VPCConfig) GetNodesWithinAddress(ipAddress string) (networkInterfaceNodes []Node, err error) {
-	var addressIPblock, networkInterfaceIPBlock *common.IPBlock
-	addressIPblock = common.NewIPBlockFromCidrOrAddress(ipAddress)
+	var addressIPblock *common.IPBlock
+	addressIPblock, err = common.NewIPBlockFromCidrOrAddress(ipAddress)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
same question here

Reply:
yes, because we don't know whether the user meant to insert an `IPBlock/Cidr`. If this failed all we know is that what he inserted does not match any legal src/dst. For example, we will get there if instead of vsi1-ky, VSI1-KY was inserted (we decided not be oblivious to cases in this stage) and the error message from `NewIPBlockFromCidrOrAddress` will be misleading. We could use some string matching to "guess" what the user meant to do, but this is surely not worth the effort for the 1st release

Reply:
this function is no longer used. Removed